src/app/Layouts/defaultPageWrapper.jsx

```diff
@@ -36,6 +47,7 @@ PageWrapper.propTypes = {
   isAmp: bool.isRequired,
   pageType: string.isRequired,
   service: string.isRequired,
+  dials: shape.isRequired,
```

Comment:
Shape requires a object of prop types to match on
```
dials: shape({
  optionalProperty: PropTypes.string,
  requiredProperty: PropTypes.number.isRequired
}),
```

Reply:
After chatting with Drew have changed this to `objectOf(string)` which is about as precise as we can be.